src/test/java/org/cactoos/list/ListEnvelopeTest.java

```diff
@@ -215,13 +215,13 @@ public void mustReturnNextElement() {
 
     private static final class StringList extends ListEnvelope<String> {
         StringList(final String... elements) {
-            super(() -> new Immutable<>(Arrays.asList(elements)));
+            super(new Immutable<>(Arrays.asList(elements)));
         }
     }
 
     private static final class MutableStringList extends ListEnvelope<String> {
         MutableStringList(final String... elements) {
-            super(() -> new LinkedList<>(Arrays.asList(elements)));
+            super(new LinkedList<>(Arrays.asList(elements)));
```

Comment:
@victornoel change `new LinkedList<>(Arrays.asList(elements))` by `new ListOf<>(elements)`.

Reply:
@fabriciofx it is not possible because it breaks the tests. I think those things are being worked on in other jobs, so I just replaced the `Arrays.asList` with `ListOf` at least.